packages/material-ui/src/SnackbarContent/SnackbarContent.d.ts

```diff
@@ -5,6 +5,7 @@ import { PaperProps } from '../Paper';
 export interface SnackbarContentProps extends StandardProps<PaperProps, SnackbarContentClassKey> {
   action?: React.ReactNode;
   message?: React.ReactNode;
+  role?: string;
```

Comment:
I think this should already be included from `PaperProps` and subsequently HTMLAttributes. If so then we should remove it here in case the HTMLAttributes narrow this type down to valid roles only

Reply:
Which file can I validate this in? I don't see role in the PaperProps in src/Paper/Paper.d.ts

https://github.com/mui-org/material-ui/blob/0f28b62cc9d5b1c070f8959abb7eb9458e42c417/packages/material-ui/src/Paper/Paper.d.ts#L4

Reply:
I think that role comes from `React.HTMLAttributes<HTMLDivElement>`.

Reply:
Oh ok, I see! I'll remove my addition to this file.